# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1265982835 -3600
#      Fri Feb 12 14:53:55 2010 +0100
# Node ID 907d470e261b9d383c91b63e950ecbecff6c9f35
# Parent  04a616348cec99594e1b40009713705c79ddb7a7
remove debug stub from interp2

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-02-12  Jaroslav Hajek  <highegg@gmail.com>
+
+	* general/interp2: Remove debug stub.
+
 2010-02-12  Jaroslav Hajek  <highegg@gmail.com>
 
 	* optimization/optimget.m: Simplify.
 
 2010-02-11  Jaroslav Hajek  <highegg@gmail.com>
 
 	* miscellaneous/parseparams.m: Extend to allow direct parsing of
 	options.
diff --git a/scripts/general/interp2.m b/scripts/general/interp2.m
--- a/scripts/general/interp2.m
+++ b/scripts/general/interp2.m
@@ -202,21 +202,16 @@ function ZI = interp2 (varargin)
       if (columns (Y) != 1)
         Y = Y.';
       endif
     endif
 
     xidx = lookup (X, XI, "lr");
     yidx = lookup (Y, YI, "lr");
 
-    if (min (xidx) <= 0)
-      save bad_data X XI
-    endif
-
-
     if (strcmp (method, "linear"))
       ## each quad satisfies the equation z(x,y)=a+b*x+c*y+d*xy
       ##
       ## a-b
       ## | |
       ## c-d
       a = Z(1:(zr - 1), 1:(zc - 1));
       b = Z(1:(zr - 1), 2:zc) - a;
